src/test/java/uk/gov/hmcts/cmc/claimstore/services/DefendantResponseServiceTest.java

```diff
@@ -64,87 +53,27 @@ public void setup() {
     }
 
     @Test
-    public void getResponseByIdShouldCallRepositoryWhenValidDefendantResponseIsReturned() {
-        //given
-        final DefendantResponse defendantResponse = createDefendantResponse(VALID_APP);
-        final Optional<DefendantResponse> result = Optional.of(defendantResponse);
-
-        when(defendantResponseRepository.getById(eq(RESPONSE_ID))).thenReturn(result);
-        //when
-        final DefendantResponse actual = responseService.getById(RESPONSE_ID);
-
-        //then
-        assertThat(actual).isEqualTo(defendantResponse);
-    }
-
-    @Test(expected = NotFoundException.class)
-    public void getByIdShouldThrowExceptionWhenDefendantResponseDoesNotExist() {
-        //given
-        final Optional<DefendantResponse> result = Optional.empty();
-
-        when(defendantResponseRepository.getById(eq(RESPONSE_ID))).thenReturn(result);
-
-        //when
-        responseService.getById(RESPONSE_ID);
-    }
-
-    @Test
-    public void getByDefendantIdShouldCallRepositoryWhenValidDefendantResponseIsReturned() {
-        //given
-        final DefendantResponse defendantResponse = createDefendantResponse(VALID_APP);
-        final List<DefendantResponse> result = Collections.singletonList(defendantResponse);
-
-        when(defendantResponseRepository.getByDefendantId(eq(DEFENDANT_ID))).thenReturn(result);
-
-        //when
-        final List<DefendantResponse> actual = responseService.getByDefendantId(DEFENDANT_ID);
-
-        //then
-        assertThat(actual).isEqualTo(result);
-    }
-
-    @Test
     public void saveShouldFinishSuccessfully() {
         //given
         final ResponseData app = SampleResponseData.validDefaults();
         final String jsonApp = new ResourceReader().read("/defendant-response.json");
         when(mapper.toJson(eq(app))).thenReturn(jsonApp);
 
         when(userService.getUserDetails(AUTHORISATION)).thenReturn(
+            new UserDetails(USER_ID, DEFENDANT_EMAIL, "Jonny", "Jones")
```

Comment:
Isn't it a good time to add builder for `UserDetails` (in next PR, not now).

Reply:
don't think there's a need, we don't manipulate users really.